homeassistant/components/recorder/__init__.py

```diff
@@ -499,13 +499,13 @@ def _commit(session, work):
         return False
 
 
-def _wait(event, message):
+def _wait(event, message, interval=15):
```

Comment:
After adding some "blocking" components like icloud, I think we should make this default even longer, maybe 30 seconds